### PR TITLE
locateme v0.2.1 (new formula)

### PR DESCRIPTION
**Pull request by beaumartinez**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

---

I did this originally on Homebrew-Cask but it was [rejected](https://github.com/caskroom/homebrew-cask/pull/25557) there. Hope this is where it belongs.

I hope this is all OK. Pushing this just before bed! ✌️

LocateMe is a nice, succinct command line tool to get your location from the built-in Mac services. It allows you to specify the format you want it in, not like other tools, so is especially handy if you're piping it.

### CODE REVIEW

File: Formula/locateme.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class Locateme < Formula
+  desc "Find your location using Apple's geolocation services"
+  homepage "http://iharder.sourceforge.net/current/macosx/locateme"
+  url "https://downloads.sourceforge.net/project/iharder/locateme/LocateMe-v0.2.1.zip"
+  sha256 "137016e6c1a847bbe756d8ed294b40f1d26c1cb08869940e30282e933e5aeecd"
+
+  def install
+    bin.install "LocateMe"
```
Comment (zmwangx) [inline Formula/locateme.rb]: We do not allow compiled binaries. You need to build it from source.

**Review by zmwangx — CHANGES_REQUESTED** [review]

**Comment by beaumartinez** [comment]
@zmwangx Pushed a change—I've never compiled Objective-C before, so I hope the way I'm doing it is OK.

**Comment by MikeMcQuaid** [comment]
@beaumartinez I was under the impression this required a signed binary to function; can you confirm if that's the case i.e. does the `brew install`d version work on your machine?

**Comment by jawshooah** [comment]
@MikeMcQuaid seems to work for me:

```bash
$ brew install --verbose --debug locateme
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FormulaLoader): loading /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/locateme.rb
==> Using the sandbox
/usr/bin/sandbox-exec -f /tmp/homebrew20161017-26111-1o817tt.sb nice /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/bin/ruby -W0 -I /usr/local/Homebrew/Library/Homebrew -- /usr/local/Homebrew/Library/Homebrew/build.rb /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/locateme.rb --verbose --debug
/usr/local/Homebrew/Library/Homebrew/build.rb (Formulary::FromPathLoader): loading /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/locateme.rb
==> Downloading https://downloads.sourceforge.net/project/iharder/locateme/LocateMe-v0.2.1.zip
==> Downloading from http://heanet.dl.sourceforge.net/project/iharder/locateme/LocateMe-v0.2.1.zip
/usr/bin/curl --fail --remote-time --location --user-agent Homebrew/1.0.7-83-g2674bfa (Macintosh; Intel macOS 10.11.6) curl/7.43.0 http://heanet.dl.sourceforge.net/project/iharder/locateme/LocateMe-v0.2.1.zip -C 0 -o /Users/jhagins/Library/Caches/Homebrew/locateme-0.2.1.zip.incomplete
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 33871  100 33871    0     0  77612      0 --:--:-- --:--:-- --:--:-- 77864
==> Verifying locateme-0.2.1.zip checksum
unzip -qq /Users/jhagins/Library/Caches/Homebrew/locateme-0.2.1.zip
==> clang -framework Foundation -framework CoreLocation LocateMe.m -o LocateMe
==> Cleaning
Fixing /usr/local/Cellar/locateme/0.2.1/bin/LocateMe permissions from 755 to 555
==> Finishing up
ln -s ../Cellar/locateme/0.2.1/bin/LocateMe LocateMe
==> Summary
🍺  /usr/local/Cellar/locateme/0.2.1: 4 files, 30.3K, built in 9 seconds

$ LocateMe
<REDACTED> +/- 65.00m (speed -1.00 mps / course -1.00) @ 10/17/16, 3:48:02 AM Eastern Daylight Time
```

Though the OS does prompt me to grant the permission to access my current location the first time I run it:

screen shot 2016-10-17 at 3 46 53 am

**Comment by ilovezfs** [comment]
Is that Sierra?

**Comment by jawshooah** [comment]
@ilovezfs Ah, nope, El Capitan. Sorry, I've got 4am brain :stuck_out_tongue_closed_eyes:

File: Formula/locateme.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  url "https://downloads.sourceforge.net/project/iharder/locateme/LocateMe-v0.2.1.zip"
+  sha256 "137016e6c1a847bbe756d8ed294b40f1d26c1cb08869940e30282e933e5aeecd"
+
+  def install
+    system ENV.cc, "-framework", "Foundation", "-framework", "CoreLocation", "LocateMe.m", "-o", "LocateMe"
+    bin.install "LocateMe"
+  end
+
+  test do
+    system "LocateMe", "-h"
```
Comment (MikeMcQuaid) [inline Formula/locateme.rb]: `system "#{bin}/LocateMe", "-h"`

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

**Comment by MikeMcQuaid** [comment]
Confirmed working on Sierra 👍

File: Formula/locateme.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+class Locateme < Formula
+  desc "Find your location using Apple's geolocation services"
+  homepage "http://iharder.sourceforge.net/current/macosx/locateme"
+  url "https://downloads.sourceforge.net/project/iharder/locateme/LocateMe-v0.2.1.zip"
+  sha256 "137016e6c1a847bbe756d8ed294b40f1d26c1cb08869940e30282e933e5aeecd"
+
+  def install
+    system ENV.cc, "-framework", "Foundation", "-framework", "CoreLocation", "LocateMe.m", "-o", "LocateMe"
+    bin.install "LocateMe"
```
Comment (zmwangx) [inline Formula/locateme.rb]: Add `man1.install "LocateMe.1"` to install the man page.

**Review by zmwangx — CHANGES_REQUESTED** [review]

**Comment by beaumartinez** [comment]
Added the manpage @zmwangx .

**Review by zmwangx — APPROVED** [review]

**Comment by beaumartinez** [comment]
@MikeMcQuaid any update on this? I've done the changes as requested.

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**